.github/workflows/manual_build.yml

```diff
@@ -60,7 +60,7 @@ jobs:
       # - name: macOS - Restore Homebrew deps from cache
```

Comment:
We probably should uncomment these steps to use Homebrew deps caching and to solve the action lint errors regarding missing `try-homebrew-cache` and `cache-homebrew-deps-restore` variables.